checkQC/qc_checkers/error_rate.py

```diff
@@ -49,6 +49,7 @@ def _qualify_error(error, lane, read):
         for lane, lane_data in qc_data.sequencing_metrics.items()
         for read, read_data in lane_data["reads"].items()
         if (qc_report := _qualify_error(read_data["mean_error_rate"], lane, read))
```

Comment:
Actually, for performance reasons, it would be better to write the conditional in the opposite order:
```python
if not read_data["is_index"] and (qc_report := _qualify_error(read_data["mean_error_rate"], lane, read)
```
This way the qc_report is only computed when `is_index` is false, see https://en.wikipedia.org/wiki/Short-circuit_evaluation for details

Reply:
Agreed :+1: